components/docker-up/go.mod

```diff
@@ -4,9 +4,12 @@ go 1.17
 
 require (
 	github.com/opencontainers/runtime-spec v1.0.2
-	github.com/rootless-containers/rootlesskit v0.14.1
+	github.com/rootless-containers/rootlesskit v1.0.0
```

Comment:
👍

Reply:
Initially this had me worried, but, after looking, [is maintenance.](https://github.com/rootless-containers/rootlesskit/compare/v0.14.6...v1.0.0)